modules/util/portal-tools-service-builder/src/main/resources/com/liferay/portal/tools/service/builder/dependencies/persistence_test.ftl

```diff
@@ -1399,7 +1399,9 @@ public class ${entity.name}PersistenceTest {
 
 				${entity.variableName}.set${entityColumn.methodName}(
 
-				<#if stringUtil.equals(entityColumn.type, "boolean")>
+				<#if stringUtil.equals(entityColumn.name, "headId") && entity.versionEntity??>
+					- RandomTestUtil.nextLong()
```

Comment:
The thing is that since we have started testing something more specific to the versioned entities context, the negative headId value indicates that is the head of an specific entry.

If we don't change that value, we would be treating as "Head" a draft entity and will be confusing during testing.

Reply:
Yes, what I mean is, should the value be the negative id of the head?